web/src/js/components/Modes/Socks.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { useAppDispatch, useAppSelector } from "../../ducks";
+import { getSpec, SocksState } from "../../modes/socks";
+import { ServerInfo } from "../../ducks/backendState";
+import { setActive } from "../../ducks/modes/socks";
+
+import { ModeToggle } from "./ModeToggle";
+
+export default function Socks() {
+    const serverState = useAppSelector((state) => state.modes.socks);
+    const backendState = useAppSelector((state) => state.backendState.servers);
+
+    const servers = serverState.map((server) => {
+        return (
+            <SocksRow
+                key={server.ui_id}
+                server={server}
+                backendState={backendState[getSpec(server)]}
+            />
+        );
+    });
+
+    return (
+        <div>
+            <h4 className="mode-title">SOCKS Proxy</h4>
+            <p className="mode-description">
+                Similar the regular proxy mode, but using SOCKS5 instead of HTTP
+                for connection establishment with the proxy.
```

Comment:
Or just:

> You manually configure your client application or device to use a SOCKSv5 proxy.

To match what we have for the regular proxy. What do you think? :)

Reply:
yes yes it's fine for me, I just copied some text from the docs but I wanted to know your opinion about that :)